packages/retail-ui-extensions/src/extension-api/navigation-api/navigation-api.ts

```diff
@@ -1,13 +1,10 @@
 export interface NavigationApiContent {
-  /** Opens a full screen with the Retail::SmartGrid::Modal extension point */
-  navigateToFullScreenModal(): void;
-
   /** Dismisses the modal highest on the stack */
   dismiss(): void;
 }
 
 /**
- * Access the navigation API to present your app in a full screen modal.
+ * Access the navigation API to dismiss your app from a full screen modal.
```

Comment:
I think the idea here is to make the navigation API something that can expand over time, not just dismiss.

Maybe something like
"Access the navigation API for navigation functionality from a full screen modal."